src/Decompiler/Analysis/InstructionUseAdder.cs

```diff
@@ -54,6 +61,26 @@ public override void VisitDefInstruction(DefInstruction def)
         {
         }
 
+        public override void VisitStore(Store store)
+        {
+            store.Src.Accept(this);
+            /* ecxOut should not be added to use list of statements like
+               `*ecxOut = ecx` */
+            if (store.Dst is Dereference)
```

Comment:
Style: consider using `//` comments over C-style `/* */` comments, to stay in line with the rest of the codebase.